Realm/Realm/Schema/PropertyType.cs

```diff
@@ -85,6 +85,11 @@ public enum PropertyType : ushort
         /// </summary>
         Decimal = 11,
 
+        /// <summary>
+        /// 128 bit UUID property.
+        /// </summary>
+        Guid = 12,
```

Comment:
Does this also mean that Lists of Guids are supported? At least I could not find any tests for it?

Reply:
They are, yes, but you're right the PR is missing tests for those - completely forgot about it 😅